src/Transport/Seam/TransportResourcesCreator.cs

```diff
@@ -17,7 +17,16 @@ public TransportResourcesCreator(ICreateTopology topologyCreator, ITopologySecti
 
         public async Task CreateQueueIfNecessary(QueueBindings queueBindings, string identity)
         {
-            if (resourcesCreated) return;
+            if (resourcesCreated)
+            {
+                return;
```

Comment:
Question: would log file contain any sort of information why we're not creating infrastructure? I.e. if someone is looking at the log to determine that topology is not created _because_ installers are disabled.